sdlib/d/gc/arena.d

```diff
@@ -751,3 +814,104 @@ unittest resizeLarge {
 	// Now full again:
 	assert(pd9.extent.hpd.full);
 }
+
+unittest resizeLargeGrow {
+	import d.gc.arena;
+	shared Arena arena;
+
+	auto base = &arena.base;
+	scope(exit) arena.base.clear();
+
+	import d.gc.emap;
+	static shared ExtentMap emap;
+	emap.tree.base = base;
+
+	import d.gc.region;
+	shared RegionAllocator regionAllocator;
+	regionAllocator.base = base;
+
+	arena.regionAllocator = &regionAllocator;
+
+	PageDescriptor makeAllocLarge(uint pages) {
```

Comment:
And looking back, the PD your eturn here also isn't necessary for anything but the extent, so returning the extent makes more sense.